11a-plots-examples-scatter.Rmd

```diff
@@ -0,0 +1,180 @@
+### Scatter Plot
+#### Overview
+This article will demonstrate how to construct a scatter plot in ggplot2. We will use a sample figure from the recent [Standard Safety Tables and Figures: Integrated Guide](https://downloads.regulations.gov/FDA-2022-N-1961-0002/attachment_1.pdf) released by the FDA as a guide. 
+
+```{r out.height = "460px", out.width='800px', echo=F, fig.cap = "Sample figure from the SSTFIG (Figure 17)"}
+knitr::include_graphics("./additional resources/f17.png")
+```
+ 
+#### Packages and Sample Data
+Let's start with accessing ADVS from the PhUSE Test Data Factory repository.
+
+```{r, message = FALSE, warning = FALSE, error = FALSE}
+# Packages
+library(ggplot2)
+library(dplyr)
+
+# Data
+advs <- haven::read_xpt(
+  paste0("https://github.com/phuse-org/TestDataFactory/",
+         "raw/main/Updated/TDF_ADaM/advs.xpt"))
+```
+
+#### Data Wrangling
+In order to make this example meaningful, data wrangling and processing steps will be covered here. In practice, you might have other considerations or processing steps that need to be considered.
+
+The first step is to obtain baseline values for each USUBJID for the parameter and analysis time point of interest. 
+```{r message = FALSE, warning = FALSE, error = FALSE}
+baseline <- advs %>%
+  filter(
+    SAFFL == "Y",
+    PARAMCD == "SYSBP",
+    ATPT == "AFTER LYING DOWN FOR 5 MINUTES",
+    TRTA != "Placebo"
+  ) %>%
+  dplyr::select(USUBJID, TRTA, BASE) %>%
+  distinct(USUBJID, .keep_all = TRUE)
+```
+
+The next step is to obtain the maximum value for each USUBJID post-baseline for the same parameter and analysis time point of interest.
+```{r message = FALSE, warning = FALSE, error = FALSE}
+post <- advs %>%
+  filter(
+    SAFFL == "Y",
+    PARAMCD == "SYSBP",
+    ATPT == "AFTER LYING DOWN FOR 5 MINUTES",
+    TRTA != "Placebo",
+    AVISIT != "Baseline"
```

Comment:
the current filter for post records contain below highlighted records (`AVISIT` is blank `VISIT` is screening/AMBUL ECG REMOVAL...) - should be excluded from analysis:
![image](https://user-images.githubusercontent.com/47867131/212405960-b14e0d1b-9e56-45a0-9c23-4bd767c21960.png)


those records can be excluded by adding `ANL01FL == "Y"`

Reply:
Thanks @kaz462 and good catch! I've updated the post filtering criteria.

Reply:
Thanks @mattkumar, looks great ! @iqis no further comments from me